static/js/base.js

```diff
@@ -119,13 +118,15 @@ function drawRobot(robot) {
 drawRobot({
     X: Number($("#xpos").val()),
     Y: Number($("#ypos").val()),
-    dir: direction
+    dir: direction,
+    type: robotType
 });
 
 function drawObjs(objs, type) {
     /* Draw objects received from the worker. */
 
     if (type === "obstacle") {
+        // obstacles = objs;
```

Comment:
It looks like you had some merge conflicts, this is probably because you originally branched off of a branch other than master, which doesn't have the same commits because we squash and merge the commits when merging branches into master. It looks like you cleaned up the code well, but there are a couple things left to clean up.